File: Python/Course.py
class Course():
    
    # atribut privat
    __kodeCourse = ""
    __namaMatkul = ""
    
    # constructor
    def __init__(self, kodeCourse = "", namaMatkul = ""):
        self.__kodeCourse = kodeCourse
        self.__namaMatkul = namaMatkul
    
    # getter setter
    def setKodeCourse(self, kodeCourse):
        self.__kodeCourse = kodeCourse
        
    def getKodeCourse(self):
        return self.__kodeCourse

    def setNamaMatkul(self, namaMatkul):
        self.__namaMatkul = namaMatkul
        
    def getNamaMatkul(self):
        return self.__namaMatkul

File: CPP/Program/Course.cpp
#include "header.hh"

// Contructor
Course::Course()
{
    kodeMatkul = "";
    namaMatkul = "";
}

// Constructor dengan atribut
Course::Course(string kodeMatkul, string namaMatkul)
{
    this->kodeMatkul = kodeMatkul;
    this->namaMatkul = namaMatkul;
}

// Getter Setter
string Course::getKodeMatkul()
{
    return kodeMatkul;
}

void Course::setKodeMatkul(string kodeMatkul)
{
    this->kodeMatkul = kodeMatkul;
}

string Course::getNamaMatkul()
{
    return namaMatkul;
}

void Course::setNamaMatkul(string namaMatkul)
{
    this->namaMatkul = namaMatkul;
}

// Destructor
Course::~Course() {}